robot_skills/src/robot_skills/arm/arms.py

```diff
@@ -902,128 +649,3 @@ def _publish_marker(self, goal, color, ns=""):
 
     def __repr__(self):
         return "Arm(side='{side}')".format(side=self.side)
-
-
-class ForceSensingArm(Arm):
-    def __init__(self, robot_name, tf_listener, get_joint_states, side):
-        """
-        constructor
-
-        :todo: Make the Arm class similar to the robot, such that it can be composed from parts
-
-        :param robot_name: robot_name
-        :param tf_listener: tf_server.TFClient()
-        :param get_joint_states: get_joint_states function for getting the last joint states
-        :param side: left or right
-        """
-        super(ForceSensingArm, self).__init__(robot_name, tf_listener, get_joint_states, side)
-
-        self.force_sensor = ForceSensor(robot_name, tf_listener, "/" + robot_name + "/wrist_wrench/raw")
-
-    def move_down_until_force_sensor_edge_up(self, timeout=10, retract_distance=0.01):
-        """
-        Move down the arm (hero specific, only joint arm_lift_joint) until the force sensor detects an edge up
-
-        A 'force_sensor.TimeOutException' will be raised if no edge up is detected within timeout
-
-        :param timeout: Max duration for edge up detection
-        :param retract_distance: How much to retract if we have reached a surface
-        """
-        # Fill with required joint names (desired in hardware / gazebo impl)
-        current_joint_state = self.get_joint_states()
-        current_joint_state['arm_lift_joint'] = 0
-        self._ac_joint_traj.send_goal(self._make_goal(current_joint_state, timeout))
-
-        self.force_sensor.wait_for_edge_up(timeout)
-        self.cancel_goals()
-
-        current_joint_state = self.get_joint_states()
-        current_joint_state['arm_lift_joint'] += retract_distance
-        self._ac_joint_traj.send_goal(self._make_goal(current_joint_state, 0.5))
-
-    def _make_goal(self, current_joint_state, timeout):
-        positions = [current_joint_state[n] for n in self.joint_names]
-        points = [JointTrajectoryPoint(positions=positions,
-                                       time_from_start=rospy.Duration.from_sec(timeout))]
-        trajectory = JointTrajectory(joint_names=self.joint_names, points=points)
-        return FollowJointTrajectoryGoal(trajectory=trajectory)
-
-
-class FakeArm(RobotPart):
```

Comment:
This entire functionality is now gone, maybe that is fair enough, but I do not know enough about it to be able to judge @MatthijsBurgh maybe you have an opinion?

Reply:
It does not appear in any challenges, robot_smach_states or robot_skills. Previously this was used to complete SERGIO's arms but right now we already support a robot without arms.